lightly/api/api_workflow_listing.py

```diff
@@ -0,0 +1,627 @@
+import time
+import warnings
+from typing import Dict, Iterator, List, Optional, Set, Tuple, Union
+
+import tqdm
+
+from lightly.api.retry_utils import retry
+from lightly.openapi_generated.swagger_client.models import (
+    DatasourceConfig,
+    DatasourceProcessedUntilTimestampRequest,
+    DatasourceProcessedUntilTimestampResponse,
+    DatasourceRawSamplesData,
+)
+from lightly.openapi_generated.swagger_client.models.datasource_raw_samples_data_row import (
+    DatasourceRawSamplesDataRow,
+)
+
+
+class _ListingMixin:
+    def download_raw_samples(
```

Comment:
can we deprecate this one as there is a `download_new_raw_samples`?

Reply:
No, we cannot, as we use the download_raw_samples in the worker. See https://github.com/lightly-ai/lightly-core/blob/33852d81379364794e4218e2c77e1e993edd5a90/onprem-docker/lightly_worker/src/lightly_worker/datasources/cloud_input_datasource.py#L94

However, we could delete/deprecate the `download_new_raw_samples`, as that one is not used in the worker. Will have a closer look at it separately.